tools/topology/topology2/cavs/platform/intel/deep-buffer.conf

```diff
@@ -0,0 +1,39 @@
+
+	Object.Pipeline {
+		deepbuffer-playback."1" {
+			index $DEEP_BUFFER_PIPELINE_ID
+
+			Object.Widget.copier.1 {
+				stream_name 'Deepbuffer Playback'
+			}
+
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name 'Playback Volume 1'
+				}
+			}
+		}
+	}
+
+	Object.PCM {
+		pcm.$DEEP_BUFFER_PCM_ID {
+			name 'DeepBuffer'
+			id $DEEP_BUFFER_PCM_ID
+			direction playback
+			playback_compatible_d0i3 1
```

Comment:
can we set it to true explicityly instead of 1. It is a bool after all

Reply:
sure,